ui/apps/platform/src/Containers/ComplianceEnhanced/Schedules/components/ReportJobs.tsx

```diff
@@ -85,14 +97,14 @@ function ReportJobs({ scanConfig }: ReportJobsProps) {
             <Toolbar>
                 <ToolbarContent>
                     <ToolbarItem alignItems="center">
-                        <ReportStatusFilter
-                            selection={filteredStatuses}
-                            onChange={onReportStatusFilterChange}
+                        <ReportStatesFilter
+                            reportStates={ensureReportStates(filteredReportStates)}
+                            onChange={onReportStatesFilterChange}
```

Comment:
Changing the term from `ReportStatus` to `ReportStates` because I believe it's more appropriate. We seem to have conflicting terms:

`Run state` is used in the proto file
`Report status` was used in the UI
`Report state` is used for the search filter

I think we need to converge on the proper term. I decided to change the `Report status` in the UI to `Report state` to be more inline with the search filter terminology. Since the term `Report state` would be used in the URL, it makes more sense to call it `Report state` in the UI too.

Reply:
Was there a discussion a few weeks ago about inconsistent distinction in UI between report and run?

For what it is worth, `Run state` seems clearer than either of the others.

Reply:
I don't recollect, but maybe it was about a `scan run` vs. a `report run`? I synced with Mansur yesterday and he mentioned something about that distinction.

Reply:
`Run state` sounds better to me as well. I'm worried about the difference in terminology between the search filter query and the UI label though 😢. I think if we used `Report run state`, it would have made things easier. I wonder if that's a good in-between.